### PR TITLE
new FCCee_o1_v02 detector model

### PR DESCRIPTION
**Pull request by oviazlo**
BEGINRELEASENOTES
- new FCCee_o1_v02 detector model
- Material budget of VTX was increased for 50%
- change of LumiCal_cell_size from 1.0 to 1.81 mm, change of LumiCall offset
- add test for FCCee_o1_v02 model, remove test for FCCee_o1_v01

ENDRELEASENOTES

### CODE REVIEW

**Comment by petricm** [comment]
@Voutsi please accept the group invite so that I can add you as a reviewer

File: FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    <define>
+            <constant name="LumiCal_cell_size" value="1.81*mm"/>
+    </define>
+    
+    <readouts>
+        <readout name="LumiCalCollection">
+	    <segmentation type="PolarGridRPhi"
+			  grid_size_r="(LumiCal_outer_radius-LumiCal_inner_radius)/32"
+			  grid_size_phi="360/32*degree"
+			  offset_r="LumiCal_inner_radius + grid_size_r/2."
```
Comment (petricm) [inline FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml]: `grid_size_r` in this line does not know what the value from two lines above

Comment (petricm) [reply FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml]: I am also not sure about the half, if you compare to [this](https://github.com/iLCSoft/lcgeo/blob/master/CLIC/compact/CLIC_o3_v14/LumiCal_o1_v01_01.xml#L8)

Comment (Voutsi) [reply FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml]: yes, that's the correct value:
 offset_r="LumiCal_inner_radius + (LumiCal_outer_radius-LumiCal_inner_radius)/64"

Comment (oviazlo) [reply FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml]: @Voutsi so, I should only change a value of offset_r to the expression above, right?

Comment (oviazlo) [reply FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml]: fixed

**Comment by Voutsi** [comment]
That's right, thanks
________________________________

**Pull request merged**